pom.xml

```diff
@@ -55,6 +55,7 @@
 		<slf4j.version>2.0.6</slf4j.version>
 		<tinyoauth2.version>0.5.1</tinyoauth2.version>
 		<zxcvbn.version>1.7.0</zxcvbn.version>
+		<appindicator.version>1.0.0-SNAPSHOT</appindicator.version>
```

Comment:
This requires JDK 19 (not `>= 19` but `== 19`), which is fine atm but we're in the process of switching to JDK 20 which will break support, as this is compiled with `--enable-preview`.

Not really a problem right now, but as long as Panama is a preview feature, this might force us to temporarily drop support for this tray icon implementation again, when updating the JDK.

I would suggest that you use JDK 20 already, since it is only a matter of days - weeks for us to make the switch.